.github/workflows/update_pyapi_stubs.yml

```diff
@@ -0,0 +1,55 @@
+name: Update Python API Stubs
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  schedule:
+  - cron: "0 2 * * 1-5"
+  workflow_dispatch:  # Allow manual triggering
+
+jobs:
+  update-stubs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Install OpenVINO nightly and stubgen
+        run: |
+          python -m pip install --pre -U openvino --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
```

Comment:
It looks like it should be triggered for each commit. How do you expect to update the stub files for the release packages?